internal/middleware/account_test.go

```diff
@@ -7,9 +7,9 @@ import (
 	"net/http/httptest"
 	"testing"
 
+	identity2 "github.com/RHEnVision/provisioning-backend/internal/identity"
```

Comment:
In tests we do have existing `testing/identity` unfortunately.

Reply:
I guess that it would make sense to make it a "stub" and we would have stubbed WithIdentity in the package, that would serve the same purpose, just for setting it up for tests?
I can take a look into that once we merge this :)